docs.json

```diff
@@ -1,72 +1,75 @@
 {
-    "$schema": "https://mintlify.com/docs.json",
-    "theme": "mint",
-    "name": "Mint Starter Kit",
-    "colors": {
-      "primary": "#2e8555",
-      "light": "#2e8555",
-      "dark": "#25c2a0"
-    },
-    "favicon": "/favicon.svg",
-    "navigation": {
-      "tabs": [
-        {
-          "tab": "Guides",
-          "groups": [
-            {
-              "group": "Getting started",
-              "pages": [
-                "index",
-                "auth",
-                "onboarding-flow"
-              ]
-            },
-            {
-              "group": "Accounts",
-              "pages": [
-                "account/accounts",
-                "account/send-funds-from-safe",
-                "account/onchain-daily-limit",
-                "account/iban-integration"
-              ]
-            },
-            {
-              "group": "Cards",
-              "pages": [
-                "cards/concepts",
-                "cards/card-order-flow",
-                "cards/pse-integration"
-              ]
-            }
-          ]
-        },
-        {
-          "tab": "API reference",
-          "groups": [
-
-            {
-                "group": "PSE API",
-                "openapi": "https://api-pse-public.gnosispay.com/api-docs/spec.json"
-              }
-          ]
-        }
-      ]
-    },
-    "logo": {
-      "light": "/static/img/logo.svg",
-      "dark": "/static/img/logo.svg"
-    },
-    "contextual": {
-      "options": [
-       "copy",
-       "view",
-       "chatgpt",
-       "claude",
-       "perplexity",
-       "mcp",
-       "cursor",
-       "vscode"
-     ]
+  "theme": "mint",
+  "name": "Gnosis Pay Documentation",
+  "colors": {
+    "primary": "#8D9746",
+    "light": "#556B2F",
+    "dark": "#8D9746"
+  },
+  "favicon": "/favicon.svg",
+  "navigation": {
+    "tabs": [
+      {
+        "tab": "Guides",
+        "groups": [
+          {
+            "group": "Getting started",
+            "icon": "play",
+            "pages": ["index", "auth", "onboarding-flow"]
+          },
+          {
+            "group": "Accounts",
+            "icon": "file-user",
+            "pages": [
+              "account/accounts",
+              "account/send-funds-from-safe",
+              "account/onchain-daily-limit",
+              "account/iban-integration"
+            ]
+          },
+          {
+            "group": "Cards",
+            "icon": "cards",
+            "pages": [
+              "cards/concepts",
+              "cards/card-order-flow",
+              "cards/pse-integration"
+            ]
+          }
+        ]
+      },
+      {
+        "tab": "API reference",
+        "groups": [
+          {
+            "group": "PSE API",
+            "openapi": "https://api-pse-public.gnosispay.com/api-docs/spec.json"
+          }
+        ]
+      }
+    ]
+  },
+  "logo": {
+    "light": "/static/img/logo.svg",
+    "dark": "/static/img/logo.svg"
```

Comment:
we need to change this to a logo that has a light color: 

<img width="364" height="138" alt="image" src="https://github.com/user-attachments/assets/06117f9b-7cc7-40cd-aaaf-0077987e7ac9" />